src/main/java/com/bootcamp/demo/model/User.java

```diff
@@ -0,0 +1,77 @@
+package com.bootcamp.demo.model;
+
+import java.util.Objects;
+import java.util.UUID;
+
+public final class User {
+    private final UUID userId;
+    private String email;
```

Comment:
Wondering if with the current approach & having a small overview of firestore, if we should have the userId provided separately, or if we should infer it from the email. What do you think? We can also leave it as such, but I was wondering if you could also see some downsides which might need to be covered (potentially not for this app, but for one targeted at managing users) through validation?

Reply:
I think the user class could have the `userId` as a field as in the current approach to be used for crud operations. The user data from the database could be assumed to be validated for our booking oriented purposes. Of course this is risky if the id is not valid. I can't quite grasp the 'big picture' downsides.
In my opinion, the validation of a new user instance should take place at Service/Controller level, maybe even in the My account part of the application when a user signs up.

Reply:
I was thinking about perhaps resulting in creating 2 users with the same email (so validation would be needed for email). But indeed, as you mentioned it is not necessarily in scope for booking (since it will not be targeted at user management).
It was meant more as a note (perhaps it would be best to include that for future comments along these lines), for possible future consideration.